backend/bot/bot.py

```
@@ -0,0 +1,35 @@
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from asgiref.sync import sync_to_async
```

Ruff (0.8.2)

`asgiref.sync.sync_to_async` imported but unused

Remove unused import: `asgiref.sync.sync_to_async`

(F401)